microsoft-edge/webview2/concepts/distribution.md

```diff
@@ -45,11 +45,12 @@ Today during the preview, the Evergreen WebView2 Runtime and Microsoft Edge Dev
 > [!IMPORTANT]
 > Do not ship WebView2 applications in production during the preview.  
 
-Please use the following workflow to ensure the Evergreen WebView2 Runtime is available.  
+Applications are recommended to ensure that the Evergreen WebView2 Runtime is installed before launch. Below is an example workflow.  
```

Comment:
I don't know if this is just me....but this somehow conveys to me that developer does this during testing, which is not what we're discussing here. 

> before you run the WebView2 application
Also it's usually the end user running the app, not the developer.

Reply:
Ensure your WebView2 application installer tests to confirm that the WebView2 Runtime (Evergreen) is installed before trying to run the WebView2 application.

Does that help address the concern?

Reply:
I'd like to avoid the `test` part and also use `Evergreen WebView2 Runtime` language which is used throughout the doc. Any particular concern with the original sentences?